packages/happy-css-modules/src/cli.ts

```diff
@@ -78,6 +78,11 @@ export function parseArgv(argv: string[]): RunnerOptions {
       default: 'info' as RunnerOptions['logLevel'],
       describe: 'What level of logs to report.',
     })
+    .option('outputFolder', {
```

Comment:
NITS: happy-css-modules values compatibility with existing tools. For this reason, I would like it to have the same option name (`outDir`) as typed-css-modules.

Can you please rename it?

Reply:
I have modified it! https://github.com/mizdra/happy-css-modules/pull/270/commits/65313471395b996962329596add2fbe5c548fc75

Reply:
Thank you!